lib/ttfunk/table/cmap/format04.rb

```diff
@@ -56,15 +55,17 @@ def self.encode(charmap)
               break
             end
 
-            start_glyph_id = new_map[a][:new]
+            start_glyph_id = new_map.fetch(a, new: 0)[:new]
```

Comment:
Unfortunately I did this long enough ago that I don't fully remember the reasoning. If memory serves I was getting `NoMethodError`s, but those may have been fixed by [this](https://github.com/prawnpdf/ttfunk/pull/51) recently merged PR.

Reply:
It seems plausible. Could you please keep amount of change to the minimum required? It would make reviews a bit easier and a bit less code to maintain.